packages/react-components/react-popover/src/components/PopoverTrigger/usePopoverTrigger.ts

```diff
@@ -22,9 +22,7 @@ import { Escape } from '@fluentui/keyboard-keys';
  */
 export const usePopoverTrigger_unstable = (props: PopoverTriggerProps): PopoverTriggerState => {
   const { children } = props;
-  const child = React.isValidElement(children)
-    ? getTriggerChild<Partial<PopoverTriggerChildProps>>(children)
-    : undefined;
+  const child = getTriggerChild<PopoverTriggerChildProps>(children);
```

Comment:
```suggestion
  const child = getTriggerChild(children);
```

Is `PopoverTriggerChildProps` required? I don't see it in other places (`Menu`, `Tooltip`).